src/assets/TestFramework/ArmDeployment.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+using System;
+
+namespace Azure.ResourceManager.TestFramework
+{
+    /// <summary> A Class representing a ArmDeployment along with the instance operations that can be performed on it. </summary>
+    public partial class ArmDeployment : ArmResource
```

Comment:
I am pretty sure this should be `ArmDeploymentResource`. Now the resource classes all have a `Resource` suffix